integration-cli/docker_cli_build_test.go

```diff
@@ -669,6 +669,64 @@ func TestBuildCacheADD(t *testing.T) {
 	logDone("build - build two images with remote ADD")
 }
 
+func TestBuildLastModified(t *testing.T) {
+	name := "testbuildlastmodified"
+	defer deleteImages(name)
+
+	server, err := fakeStorage(map[string]string{
+		"file": "hello",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+
+	var out, out2 string
+
+	if _, out, err = buildImageWithOut(name,
+		fmt.Sprintf("FROM busybox\nADD %s/file /\nRUN ls -le /file", server.URL),
```

Comment:
Just use backtick multiline strings, they look better and we'd have consistency across test methods.